src/components/FeaturedImage/FeaturedImage.module.scss

```diff
@@ -1,5 +1,5 @@
 .featured-image {
-  padding: 2rem;
```

Comment:
Each post image already has a border so having this padding doubles it.

Reply:
I think @theodesp and I took two different approaches to adding the border. I used padding on the header component's image and @theodesp used an actual border for the `Posts` component. I honestly don't remember why I went with `padding` over `border`, but removing the padding here does result in a missing border on a few different things (i.e. the header component's images on single post views and the `Project` components images).

I'm fine removing the `padding` in favor of `border`, but in that case I'd recommend moving the `border` style into this component so that we only have to style it once.

**Side note:** The image of Austin from Brian has a border baked in. We'll need to remove that since we want the borders to be consistently applied in CSS.

Reply:
Oh I see. Good call out. I've always felt `padding` worked better with the box model for borders like this, but don't really have any basis behind it. I'm good with either approach!

Reply:
I restored padding in the featured image.